README.md

```diff
@@ -62,6 +62,7 @@ If you want to contribute, please read [CONTRIBUTING.md](./CONTRIBUTING.md).
 ## Services
 
 * [WorkflowCI](https://www.workflowci.com) – IFTTT for developers (freemium). Integrates with Slack, GitHub, CircleCI, Google Cloud Build.
+* [StackStorm](https://github.com/StackStorm/) - StackStorm Chatops in a form of packages and docker image. The package includes hubot, hubot-stackstorm and pre-installed adapters for many Chat services.
```

Comment:
could we specify that it is a self-hosted platform + which chat providers it supports? e.g.

> self-hosted platform for integration and automation across services and tools. Integrates through Hubot with Slack, Microsoft Teams, Mattermost, Rocket.Chat, Cisco Spark.